### ISSUE TITLE
fix: 1-org/envs/shared/projects.tf interconnect and dns labels need alignment to changes in #1199

### ISSUE THREAD

**Issue by fmichaelobrien** [terraform-google-modules/terraform-example-foundation#1231]
### TL;DR

tef minor bug for label changes in https://github.com/terraform-google-modules/terraform-example-foundation/pull/1199/files#diff-d6697e7c916ba73d6ae87ff4b1ce67cabc9b9738ab31c9ba582e2a3218982838L279

https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/1-org/envs/shared/projects.tf#L253
match
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/1-org/envs/shared/projects.tf#L237
```
-    application_name  = "org-dns-hub"
+    application_name  = "org-net-dns"
```

and
```
-net-interconnect
```

just like in 
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/1-org/envs/shared/projects.tf#L295

### Terraform Configuration

```hcl
cloud shell
```

### Terraform Version

```sh
1.3.10
```

### Additional information

part of upstream sync in 387
pr assigned to fmichaelobrien

**Comment by fmichaelobrien** [comment]
```
fmichaelobrien-macbookpro:terraform-example-foundation fmichaelobrien$ git status
On branch gh1231-1-org-labels
Your branch is up to date with 'origin/gh1231-1-org-labels'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   1-org/envs/shared/projects.tf

no changes added to commit (use "git add" and/or "git commit -a")
fmichaelobrien-macbookpro:terraform-example-foundation fmichaelobrien$ git diff
diff --git a/1-org/envs/shared/projects.tf b/1-org/envs/shared/projects.tf
index 681d857..4ba625e 100644
--- a/1-org/envs/shared/projects.tf
+++ b/1-org/envs/shared/projects.tf
@@ -176,7 +176,7 @@ module "interconnect" {
 
   labels = {
     environment       = "network"
-    application_name  = "org-interconnect"
+    application_name  = "org-net-interconnect"
     billing_code      = "1234"
     primary_contact   = "example1"
     secondary_contact = "example2"
@@ -250,7 +250,7 @@ module "dns_hub" {
 
   labels = {
     environment       = "network"
-    application_name  = "org-dns-hub"
+    application_name  = "org-net-dns"
     billing_code      = "1234"
     primary_contact   = "example1"
     secondary_contact = "example2"

```

**Comment by fmichaelobrien** [comment]
closed based on review discussion in #1232
